ratatool-examples/src/main/scala/com/spotify/ratatool/examples/scalacheck/GenExample.scala

```diff
@@ -96,14 +96,10 @@ object ExampleAvroGen {
        * Set dependent fields based on Schema criteria. This is done in a single amend with
        * a single gen to ensure values are consistent per record
        */
-      .amend(intGen)(m => i => {
-        m.setIndependentIntField(i)
-        m.setDependentIntField(dependentIntFunc(i))
-      })
-      .amend(stringGen)(m => s => {
-        m.setIndependentStringField(s)
-        m.setDependentEnumField(dependentEnumFunc(s))
-      })
+      .amend(intGen)(_.setIndependentIntField,
```

Comment:
Bit of a nitpick but can we add the first function on a newline here, makes it more clear when reading the examples that it's doing two things